From e748988284621d0f7c03293b93652faefb3b08c9 Mon Sep 17 00:00:00 2001
From: Liav A <liavalb@gmail.com>
Date: Fri, 24 Dec 2021 18:38:15 +0200
Subject: [PATCH] Use /bin/bash as the shebang

/bin/sh is a symlink to Shell, which is not sh-compatible.
---
 pfetch | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pfetch b/pfetch
index d47b878..d8f98ba 100755
--- a/pfetch
+++ b/pfetch
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 #
 # pfetch - Simple POSIX sh fetch script.
 
-- 
2.36.1

